lib/client/conntest/database/database.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package database
+
+import (
+	"github.com/gravitational/trace"
+)
+
+// PingParams contains the required fields necessary to test a Database Connection.
+type PingParams struct {
+	// Host is the hostname of the Database (does not include port).
+	Host string
+	// Port is the port where the Database is accepting connections.
+	Port int
```

Comment:
These are proxy's host/port, not database's? Let's make this clear in the godocs. Or better yet rename to `ProxyHost` and `ProxyPort`.

Reply:
From the client / `Ping` PoV, there's no proxy
Just a host and a port the driver is connecting to.

Adding the `Proxy` suffix here is, IMO, no necessary and increases the cognitive load when debugging this file.
> What Proxy? The ALPN Proxy? The Teleport's proxy service?

It also decreases the probability of moving this Database Pinger into a separate library.